src/_Classics_/caesar_cipher/caesarCipher.test.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-else-return */
+const caesarCipher = require('./caeserCipher');
+
+const compareString = (str1, str2) => {
+  if (str1 === str2) {
```

Comment:
this can be done in one line

Reply:
In fact, it can be done directly in the test, string1 === string2 is the same as expected(string1).toBe(string2)

Reply:
Hey I have edited all the expect().toBe() as per the comment of @fcosueza. Review it again.

Reply:
The tests seems good to me, just be sure that tests cover 100% of the code. You can check this with 'jest -- --coverage'. About the algorithm file, just follow @TheSTL indications in:

 https://github.com/knaxus/problem-solving-javascript/pull/57#issuecomment-540399877